projects/shr-mediator/src/botswana/services/ipms.service.ts

```diff
@@ -391,14 +415,39 @@ export class IpmsService {
             : undefined;
 
         if (labOrderId && labOrderId.value) {
-          const options: AxiosRequestConfig = {
+        
+
+          const formerOptions: AxiosRequestConfig = {
```

Comment:
Are these still former ?

Reply:
formerOptions is for fetching the previous Service Request resource. options (current options) is for fetching the task resource using include Search based on the retrieved SR resource.